brands/tourism/hotel.json

```diff
@@ -669,6 +794,15 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|Westin": {
+    "tags": {
+      "brand": "Westin",
+      "brand:wikidata": "Q1969162",
+      "brand:wikipedia": "en:Westin Hotels & Resorts",
+      "name": "Westin",
```

Comment:
Individual hotel locations are named “The Westin” (“Le Westin” in Québec), whereas the company name leaves out the article.

Reply:
Fixed